packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -580,10 +576,12 @@ export default {
                     address: this.address,
                     customer: this.customer,
                     isCheckoutMethodDelivery: this.isCheckoutMethodDelivery,
+                    isCheckoutMethodDineIn: this.isCheckoutMethodDineIn,
                     time: this.time,
                     userNote: this.userNote,
                     geolocation: this.geolocation,
-                    asap: this.hasAsapSelected
+                    asap: this.hasAsapSelected,
```

Comment:
```suggestion
                    isAsap: this.hasAsapSelected,
```
just a nitpick. ignore if you think that `asap` is better

Reply:
I think `asap` is the property name sent to the server. And this is where not having trailing commas is bad, this line wasn't added as part of my work. It's an existing line.